.github/CHANGELOG.md

```diff
@@ -2,6 +2,43 @@
 
 <h3>New features since last release</h3>
 
+* PennyLane can now perform quantum circuit optimization using the
+  top-level transform `qml.compile`.
+  [(#1475)](https://github.com/PennyLaneAI/pennylane/pull/1475)
+
+  For example, take the following quantum function:
+
+  ```python
+  def qfunc(x, y, z):
+      qml.Hadamard(wires=0)
+      qml.Hadamard(wires=1)
+      qml.Hadamard(wires=2)
+      qml.RZ(z, wires=2)
+      qml.CNOT(wires=[2, 1])
+      qml.RX(z, wires=0)
+      qml.CNOT(wires=[1, 0])
+      qml.RX(x, wires=0)
+      qml.CNOT(wires=[1, 0])
+      qml.RZ(-z, wires=2)
+      qml.RX(y, wires=2)
+      qml.PauliY(wires=2)
+      qml.CY(wires=[1, 2])
+      return qml.expval(qml.PauliZ(wires=0))
+  ```
+
+  ```pycon
+  >>> dev = qml.device('default.qubit', wires=[0, 1, 2])
+  >>> compiled_qfunc = qml.compile()(qfunc)
+  >>> compiled_qnode = qml.QNode(compiled_qfunc, dev)
+  >>> print(qml.draw(compiled_qnode)(0.2, 0.3, 0.4))
+   0: ──H───RX(0.6)───────────────────┤ ⟨Z⟩
+   1: ──H──╭X─────────────────╭CY─────┤
+   2: ──H──╰C────────RX(0.3)──╰CY──Y──┤
+  ```
+
+  The ``qml.compile`` transform is flexible and accepts a custom pipeline
+  of quantum function transforms (you can even write your own transform!).
```

Comment:
It would be useful to see an example of passing a custom pipeline as well as the default behaviour given above. But maybe the changelog entry doesn't need that much detail?

Reply:
Good idea, I'll add an example, and this will also help clarify what the default behaviour is :+1:

Reply:
```suggestion
  of tape and quantum function transforms (you can even write your own!).
```